services/rds.py

```diff
@@ -1,95 +1,290 @@
+        ]
+        recovery_point_creation_dates = sorted([i["CreationDate"] for i in recovery_points])
+        if datetime.datetime.now(tz=tzlocal()) - recovery_point_creation_dates[-1] < datetime.timedelta(days=1):
+            compliant_resources.append(cluster["DBClusterArn"])
+        else:
+            non_compliant_resources.append(cluster["DBClusterArn"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def aurora_mysql_backtracking_enabled():
+    compliant_resources = []
+    non_compliant_resources = []
+    clusters = client.describe_db_clusters()["DBClusters"]
+
+    for cluster in clusters:
+        if cluster["Engine"] == "aurora-mysql" and cluster.get("EarliestBacktrackTime", None) == None:
+            non_compliant_resources.append(cluster["DBClusterArn"])
+        else:
+            compliant_resources.append(cluster["DBClusterArn"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def db_instance_backup_enabled():
+    compliant_resources = []
+    non_compliant_resources = []
+    clusters = client.describe_db_clusters()["DBClusters"]
+
+    for cluster in clusters:
+        if cluster.get("BackupRetentionPeriod", None) != None:
+            compliant_resources.append(cluster["DBClusterArn"])
+        else:
+            non_compliant_resources.append(cluster["DBClusterArn"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def rds_cluster_auto_minor_version_upgrade_enable():
+    compliant_resources = []
+    non_compliant_resources = []
+    clusters = client.describe_db_clusters()["DBClusters"]
+
+    for cluster in clusters:
+        if cluster.get("AutoMinorVersionUpgrade", None) == True:
+            compliant_resources.append(cluster["DBClusterArn"])
+        else:
+            non_compliant_resources.append(cluster["DBClusterArn"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def rds_cluster_default_admin_check():
+    compliant_resources = []
+    non_compliant_resources = []
+    clusters = client.describe_db_clusters()["DBClusters"]
+
+    for cluster in clusters:
+        if cluster.get("MasterUsername", None) not in ["admin", "postgres"]:
```

Comment:
MasterUsername은 RDS cluster 생성 시 required field이기 때문에 None이 되는 경우가 없습니다.
```suggestion
        if cluster["MasterUsername"] not in ["admin", "postgres"]:
```